bokeh/models/mappers.py

```diff
@@ -99,7 +99,7 @@ class CategoricalMapper(Mapper):
 
     '''
 
-    factors = Either(Seq(String), Seq(Tuple(String, String)), Seq(Tuple(String, String, String)), default=None, help="""
+    factors = Either(Seq(String), Seq(Tuple(String, String)), Seq(Tuple(String, String, String)), help="""
```

Comment:
As long as we are making a `Factor` property type, could make `Factors` (or `FactorSeq`)